src/main/java/Duke.java

```diff
@@ -6,5 +13,49 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?\n");
+
+        Scanner input = new Scanner(System.in);
+        ArrayList<String> list = new ArrayList<>();
```

Comment:
Plural form should be used when naming collection of arrays. Maybe this can be named as lists instead.